### ISSUE TITLE
"pac paportal list" throws exception

### ISSUE THREAD

**Issue by DennisValente** [microsoft/powerplatform-vscode#63]
When running command `pac paportal list` an exception is thrown.

> Connected to...Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id:12afe9c5-d072-4658-aa36-0cec5c53fc9b

- Telemetry is enabled so hopefully there is more details in your logs
- I proceeded to clear the auth list, recreate the auth profile, and try again. No luck.
- I am connecting to Gov cloud, so URLs are of the `https://[domain].crm9.dynamics.com` variety

**Comment by davidjenni** [comment]
@DennisValente this was an improperly handled AAD error, seemingly due to AAD's ConditionalAccess requirement:
AADSTS70043: The refresh token has expired or is invalid due to sign-in frequency checks by conditional access.
CC: @ameyaapte1 

Also, the pac CLI today has a limitation where its connectivity to sovereign clouds isn't fully functional. Full support for sov clouds is in the works, though. ETA: July

**Comment by DennisValente** [comment]
Thanks for the explanation @davidjenni!

> Also, the pac CLI today has a limitation where its connectivity to sovereign clouds isn't fully functional. Full support for sov clouds is in the works, though. ETA: July

All three of the `pac paportal` commands worked for me a few days before I submitted this issue. So either there was a release within that timeframe that broke those commands (possibly solely for sovereign clouds) (and `pac paportal download` had the same error message) or those commands are not currently reliable as they work merely sometimes.

Seeing as the Power Apps CLI and this extension are Preview, it's all good. The ETA of July works, as there are workarounds albeit not automatable ones.

**Comment by DennisValente** [comment]
A colleague is successful at connecting to the same portal website as I reported. Since the `pac paportal` commands still error for me, I once again performed a `pac auth clear` and `pac auth create`. The `pac paportal` commands are working once again. 

Before creating this issue, as a troubleshooting step, I did clear the auth list, reauth'd, and the paportal commands were unsuccessful. However, doing it again now allows me to successfully run the `pac paportal` commands.

This is no longer an issue.
